### DISCUSSION TITLE
发音评估一直出错

### DISCUSSION THREAD

**Discussion by zhaoyl123** [ZuodaoTech/everyone-can-use-english#869] [Q&A]
![Snipaste_2024-07-22_20-25-46](https://github.com/user-attachments/assets/b27ae1ed-1255-4acd-bfb8-0f723e378229)

**Comment by an-lee** [comment]
确保 Enjoy 是最新版 v0.3.2，下载地址： https://1000h.org/enjoy-app/install.html

软件设置 -> 高级设置 -> API 设置

点 `修改`，API 地址改为 `https://api.getenjoyapp.com` ，点 `保存`
